ITR/interfaces.py

```diff
@@ -704,15 +845,16 @@ class TemperatureScoreControls(PintModel):
     target_end_year: int
     projection_start_year: int
     projection_end_year: int
-    tcre: Quantity['delta_degC']
-    carbon_conversion: Quantity['CO2']
-    scenario_target_temperature: Quantity['delta_degC']
+    tcre: quantity('delta_degC')
```

Comment:
why do we need to redefine a class as fundamental as `Quantity`?

Reply:
Yet another mystery related to GitHub state.  In the current source code tcre is defined at line 892.

Reply:
no, this line is still there, see this link: https://github.com/os-climate/ITR/pull/159/files#diff-68a025a951c020c1fdcc98d3ac4d17f0952dfba31676fc770d03d7b04394d062R497

Reply:
Quantity is a pint-ish thing and it does not do any Pydantic validation.  quantity is defined as a function that does validation, returning a properly pint type (Quantity) that has been validated against the type of the input (in this case delta_degC).